data/entities.yaml

```diff
@@ -1223,6 +1223,17 @@
       all the nearby actors other than oneself.
   properties: [pickable]
   capabilities: [meet, meetAll]
+- name: atlas
+  display:
+    attr: device
+    char: 'a'
+  description:
+    - |
+      An atlas enables the `waypoint` command: `waypoint : Text -> Int -> Cmd (Int * (Int * Int))`.
```

Comment:
Can I interest you in taking this opportunity to switch to something like `waypoints : Text -> rec l. Unit + (Int * Int) * l` instead?  The current type has always seemed to me like a strange workaround for not having lists.  I know that might be a bunch of work to convert everything, but I'm willing to help.

Reply:
I'm willing to switch to lists, if you can do the heavy lifting :)

Reply:
OK, let me work on a PR.